spec/rails_helper.rb

```diff
@@ -6,6 +6,7 @@
 require 'shoulda-matchers'
 require 'capybara/rspec'
 require 'coveralls'
+require 'webmock/rspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.